### PR TITLE
Change ping to use informative embed colors

### PR DESCRIPTION
**Pull request by DaRealCodeWritten**

### CODE REVIEW

**Comment by DaRealCodeWritten** [comment]
Ready for review unless anyone has anything to add?

File: commands/ping.py [hunk 1]
```diff
@@ -12,12 +12,25 @@ def __init__(self, bot):
...
-            await ctx.send(repr(e))
+        colors = {
+            299: discord.Color.green(),
+            300: 0xFFFF00,
+            500: discord.Color.orange(),
+            1000: discord.Color.red()
+        }
+        m = await ctx.send("Pinging...")
+        clr = None
+        for png, color in colors.items():
```
Comment (wakfi) [inline commands/ping.py:23]: `int((snowflake_time(m.id) - snowflake_time(ctx.message.id)).microseconds / 1000)`
How about we calculate this once

File: commands/ping.py [hunk 2]
```diff
@@ -12,12 +12,25 @@ def __init__(self, bot):
...
+            if png == 299 and int((snowflake_time(m.id) - snowflake_time(ctx.message.id)).microseconds / 1000) < png:
```
Comment (wakfi) [inline commands/ping.py]: `int((snowflake_time(m.id) - snowflake_time(ctx.message.id)).microseconds / 1000)`  
How about we calculate this once

**Comment by wakfi** [comment]
Retitling PR based on ping colors as it will be the commit message. The error handler fixes can be noted in the commit message body

**Comment by DaRealCodeWritten** [comment]
Thanks for the assistance on this wakfi, your changes look good

**Review by wakfi — APPROVED** [review]
Works well

**Pull request merged**